README.md

```diff
@@ -12,19 +12,27 @@
     Eclipse Foundation. All other trademarks are the property of their respective owners.
 
 -->
-# Nexus Repository Conan Format
+# Nexus Repository JFrog Conan Format
```

Comment:
These changes related to the [PR#1](https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/1)